.github/workflows/markdown.yml

```diff
@@ -3,15 +3,19 @@ on:
   push:
     branches:
     - master
+    paths:
+    - '**.md'
   pull_request:
     branches:
     - master
+    paths:
+    - '**.md'
```

Comment:
We can't skip in this way because there are links to other source files.

Reply:
How about 'docs/*'?